src/maven/msmsscoringsettings.cpp

```diff
@@ -0,0 +1,25 @@
+#include "msmsscoringsettings.h"
+
+MSMSScoringSettingsDialog::MSMSScoringSettingsDialog(QWidget *parent) :
+	QDialog(parent) { 
+        setupUi(this);
+		setModal(false);
+}
+
+void MSMSScoringSettingsDialog::setLipidClassAdductFile(){
+
+    const QString name = QFileDialog::getOpenFileName(this,
+                "Select Lipid Class/Adduct Parameters ", ".",tr("Comma-Separated Values (*.csv)"));
+
```

Comment:
`TODO`: test what happens if the filechooser dialog is cancelled

Reply:
If the dialog is cancelled, the selected file path becomes an empty string, replacing whatever file path was there. This seems like reasonable behavior.